domains/measurements/Measurements.API/API/Startup.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Linq;
+using API.Options;
+using API.Shared.Swagger;
+using Asp.Versioning;
+using EnergyOrigin.TokenValidation.Options;
+using Microsoft.Extensions.Options;
+using Swashbuckle.AspNetCore.SwaggerGen;
+using System.Text.Json.Serialization;
+using System.Text.Json;
+using FluentValidation;
+using FluentValidation.AspNetCore;
+using Asp.Versioning.ApiExplorer;
+using EnergyOrigin.TokenValidation.Utilities;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace API;
+
+public class Startup
+{
+    private readonly IConfiguration _configuration;
+
+    public Startup(IConfiguration configuration)
+    {
+        _configuration = configuration;
+    }
+
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddHealthChecks();
+
+        services.AddControllers().AddJsonOptions(options =>
+        {
+            options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter());
+            options.JsonSerializerOptions.PropertyNamingPolicy = JsonNamingPolicy.CamelCase;
```

Comment:
Is this the same naming policy throughout all our subsystems?

Reply:
No - transfer does not at least. I removed the 'options.JsonSerializerOptions.PropertyNamingPolicy = JsonNamingPolicy.CamelCase;' line